frontend/src/components/indices/IndicesValidate.js

```diff
@@ -116,7 +125,7 @@ const IndicesValidate = ({ token, indicesTotalCount, isFetching, list, validate
   const onSubmit = () => {
     setValidationLoading(true)
     const data = serialize(formData)
-    validate(data)
+    validateCallback(data)
```

Comment:
I'm not sure why `validateCallback` is defined as a callback, since it is being used like a simple function here. Ideally, `onSubmit` should be a callback.

Reply:
I just called it that because I used useCallback.